### ISSUE TITLE
ORT und PLZ über gespeicherte Adressdaten raten/vorschlagen

### ISSUE THREAD

**Issue by RalphBike** [weg-li/weg-li#340]
Hi,

für den Fall, dass GPS nicht klappt: gibt es einen Weg, sich die ewige Eingabe von Stadt (immer die gleiche bei mir) und PLZ sehr oft die gleichen und immer die gleichen pro Adresse einzugeben? Straße kriege ich hin, Stadt auch, nervt aber. Wirklich ätzend ist die Eingabe der PLZ, weil man die selten weiß und dann nachschlagen muss.

Ich finde den Ansatz bei "Kennzeichen => Marke => Farbe" sehr charmant, ließe sich der Ansatz auch auf "Adressen => Stadt => PLZ" anwenden? 

Danke!

**Comment by phoet** [comment]
Ich verstehe das mit dem Ansatz nicht. Marke und Farbe sind optionale Angaben.

Besonders die Postleitzahl ist extrem wichtig, weil darüber die Zuordnung der zuständigen Stelle passiert.

Falls die Bilder kein GPS enthalten solltest Du den Map-Picker nutzen, da dieser alle Informationen inkl. der Geo-Koordinaten korrekt zuweist.

**Comment by RalphBike** [comment]
Hi, 

relativ einfach, und unabhängig von optional/mandatory:
bei Kennzeichen/Marke/Farbe wird über den ersten, eindeutigen Parameter der Rest geraten.

Das gleiche Verfahren könnte man auch bei der Adresse anwenden: Straße und Hausnummer eingeben, dabei bekannte ähnliche Adressen zum Vervollständigen anbieten und wenn es ein Match gibt, die beiden weiteren Parameter Ort und PLZ eintragen.

Das erleichtert die Eingabe für die Leute, die die Straße schon kennen, erspart zwei weitere Eingaben und erfordert auch keinen Map-Picker.

Alles basierend auf der Annahme, dass die Heavy-User ihre üblichen Verdächtigen meist an den gleichen Orten treffen.

**Comment by phoet** [comment]
Die Heavy-User nutzen weitestgehend Fotos mit EXIF GPS Daten.

Ich würde diese Funktionalität relativ weit hinten in der Prioritäten-Liste einsortieren.

**Comment by RalphBike** [comment]
Jo, mache ich auch, nur klappt das nicht im jedem Fall, weil das GPS nicht immer zuverlässig oder auch mal aus ist.

**Comment by dbermbach** [comment]
Manchmal habe ich das Problem mit dem nicht funktionierenden GPS auch. Eine einfache Änderung für den Mappicker würde aber die Usability verbessern: Bisher centered der Mappicker immer auf die angegebene ladungsfähige Adresse. Das ist bei mir die Arbeitsadresse und darum recht weit weg von den Orten, wo ich meist Meldungen erstelle. Könnte man die Init-Adresse für den Mappicker, wenn keine GPS-Daten in EXIF sind wahlweise
a) auf einen konfigurierbaren Ort oder
b) auf den Ort der letzten Meldung legen?

Das würde den Scrollaufwand bei der Nutzung des Mappickers deutlich verringern.

**Comment by g6094199** [comment]
ueberschneidug mit diesem [issue](https://github.com/weg-li/weg-li/issues/771)

**Comment by phoet** [comment]
Eine einfache Hilfestellung wäre es Orte auswählbar zu machen, zb letzte Meldung oder Meldung für dieses Fahrzeug
